verification/curator-service/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "curator-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "scripts": {
+    "start": "node dist/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
other repo is using MIT -- should we use that here?

Reply:
Yep! The license file in this repo is MIT, too. Thanks for the catch.